### ISSUE TITLE
The block must respect moodle/grade:manage capability

### ISSUE THREAD

**Issue by mudrd8mz** [DegrangeM/moodle-block_grades_sorter#2]
The block does not check the user's permissions to modify the course gradebook. It only checks if the user has some of the blocks editing capabilities on the page, which is not enough - there can be users allowed to change blocks, who should not touch the gradebook. And vice versa - there can be users who cannot change the blocks, but should be able to configure the gradebook.

I would suggest to perform the same checks that are done in the grade/edit/tree/index.php and particularly offer the block features only to users with the `'moodle/grade:manage` capability.

In other words, split editing the block and editing the gradebook.

**Comment by DegrangeM** [comment]
Fixed thanks.
